test/testvm.py

```diff
@@ -83,10 +83,10 @@ def stop(self):
 
     def wait_ssh(self):
         """Try to connect to self.address on port 22"""
-        tries_left = 5
+        tries_left = 60
         while (tries_left > 0):
             sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            sock.settimeout(2)
+            sock.settimeout(1)
```

Comment:
this is now already in master: https://github.com/cockpit-project/cockpit/commit/878cd8547335171fffd7de2b8b9a3b3a52bdf606